src/main/groovy/io/seqera/wave/configuration/BuildConfig.groovy

```diff
@@ -106,6 +106,12 @@ class BuildConfig {
     @Value('${wave.build.retry-attempts:0}')
     int retryAttempts
 
+    @Value('${wave.build.max_conda_file_size:100000}')
```

Comment:
Let's stick on kebab-case instead of snake_case

Reply:
done